renovate.json

```
@@ -4,6 +4,7 @@
     ":semanticCommits"
   ],
   "timezone": "Europe/Zurich",
+  "branchPrefix": "renovate_",
```

```suggestion
  "branchPrefix": "renovate-",
```

Hostnames cannot contain underscores.

---

but then we also need to remove the `@`, eg from: `renovate/@adobe-fixes`

---

[Yes](https://github.com/adobe/helix-shared/pull/388/commits/292b9310572f7fd2ad3f259e2c4100bd4c7e7cc7)